### DISCUSSION TITLE
Sum of json array lengths

### DISCUSSION THREAD

**Discussion by lubrst** [directus/directus#15825] [Archive]
Following SQL query (PostgreSQL) returns the sum of json array lengths:

`SELECT SUM(json_array_length(collection.json_array)) FROM collection`

Is such a query possible via the api?

I tried following aggregation:

```
{
    "aggregate[sum]": "count(json_array)"
}
```

But following error occurs:

```
{
    "errors": [
        {
            "message": "fields is not iterable",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ]
}
```

I don't know if it is a bug or currently not supported?

**Comment by br41nslug** [comment]
It currently not supported (or implemented) to do json queries unfortunately.
